neurom/features/__init__.py

```diff
@@ -60,9 +60,6 @@ def register_neurite_feature(name, func):
         func: single parameter function of a neurite.
 
     """
-    if name in NEURITEFEATURES:
-        raise NeuroMError('Attempt to hide registered feature %s' % name)
-
```

Comment:
Why this is removed? I was thinking we want to avoid overwriting existing features (same behaviour as with the decorator).

Reply:
Because this function calls `_register_feature` which already performs this check.

Reply:
Right, I missed that call 👍